### DISCUSSION TITLE
tweaking an address in the TSML

### DISCUSSION THREAD

**Discussion by RBIWebsite** [code4recovery/12-step-meeting-list#469] [Q&A] [answered]
I have  a meeting that occurs in a shopping mall. There are 5-6 buildings with multiple storefronts that all use the same address.  GPS takes those using it to the driveway, which  is 1/4 mile and 3 buildings away from the actual meeting location.
Can I add actual GPS coordinates to take users directly to the door where the meeting is happening? If not, any other suggestions?

**Comment by tech2serve** [comment, marked as answer]
There isn’t really an easy way to do this other that to pick an address closest to the actual meeting location. We do have an open issue to improve the ability to add specific coordinates, but not sure when that will get worked on, if ever.

I travel around a lot, and have run across the issue you’re describing. Don’t each of the storefronts have a suite number/letter that can be used to identify it in the notes, or perhaps has part of the address?

Sometimes I’ve found you can use Google’s Plus Code to trick the address. Go to maps.google.com and enter the address in question, perhaps a marker, and then try to get a closer address. Sometimes using Street View will reveal a closer usable address that can be entered. 

Another thing you may want to do is to report the issue to Google, especially if each of the buildings has some sort of identifier (A, B, C?)

**Comment by RBIWebsite** [comment]
Thanks so much for your reply. My first swag was to find/use a suite # as well. I called the location to get it, & was told, “ I dunno”……..😳
The front door pic is on Google, & there isn’t an identifier on it.  Some meetings are more anonymous than others……..I would have knocked on every door I could find in my past if I was looking for free beer…….just saying!🤗

**Reply by tech2serve** [reply]
Unfortunately, this is the best I have for now. Perhaps you can describe (probably already are) how many units down from entrance to the lot?

**Comment by RBIWebsite** [comment]
I appreciate the reply for sure.
I’m frustrated that the home group members behave like this is someone else’s problem, even though they brought it to me and asked what I could do to help.
I even called the facility and ask them for a suite #, & got a short “ I dunno” answer. 
If they don’t care enuff to put forth some effort, I can’t care for them. I also suggested they buy a circle & triangle logo to hang on the door while the meeting is happening, & got a deer-in-the-headlights stare….
